docs/docsite/rst/inventory_guide_iocage_aliases.rst

```diff
@@ -0,0 +1,208 @@
+
+.. code-block:: console
+
+   shell> iocage start ALL
+   No default gateway found for ipv6.
+   * Starting 052b9557
+     + Started OK
+     + Using devfs_ruleset: 1000 (iocage generated default)
+     + Configuring VNET OK
+     + Using IP options: vnet
+     + Starting services OK
+     + Executing poststart OK
+     + DHCP Address: 10.1.0.137/24
+   No default gateway found for ipv6.
+   * Starting 1c11de2d
+     + Started OK
+     + Using devfs_ruleset: 1001 (iocage generated default)
+     + Configuring VNET OK
+     + Using IP options: vnet
+     + Starting services OK
+     + Executing poststart OK
+     + DHCP Address: 10.1.0.146/24
+   No default gateway found for ipv6.
+   * Starting 9d94cc9e
+     + Started OK
+     + Using devfs_ruleset: 1002 (iocage generated default)
+     + Configuring VNET OK
+     + Using IP options: vnet
+     + Starting services OK
+     + Executing poststart OK
+     + DHCP Address: 10.1.0.115/24
+   Please convert back to a jail before trying to start ansible_client
+
+List the jails:
+
+.. code-block:: console
+
+   shell> iocage list -l
+   +-----+----------+------+-------+------+-----------------+--------------------+-----+----------------+----------+
+   | JID |   NAME   | BOOT | STATE | TYPE |     RELEASE     |        IP4         | IP6 |    TEMPLATE    | BASEJAIL |
+   +=====+==========+======+=======+======+=================+====================+=====+================+==========+
+   | 207 | 052b9557 | off  | up    | jail | 14.2-RELEASE-p3 | epair0b|10.1.0.137 | -   | ansible_client | no       |
+   +-----+----------+------+-------+------+-----------------+--------------------+-----+----------------+----------+
+   | 208 | 1c11de2d | off  | up    | jail | 14.2-RELEASE-p3 | epair0b|10.1.0.146 | -   | ansible_client | no       |
+   +-----+----------+------+-------+------+-----------------+--------------------+-----+----------------+----------+
+   | 209 | 9d94cc9e | off  | up    | jail | 14.2-RELEASE-p3 | epair0b|10.1.0.115 | -   | ansible_client | no       |
+   +-----+----------+------+-------+------+-----------------+--------------------+-----+----------------+----------+
+
+Set notes. The tag *alias* will be used to create inventory aliases:
+
+.. code-block:: console
+
+   shell> iocage set notes="vmm=iocage_02 project=foo alias=srv_1" 052b9557
+   notes: none -> vmm=iocage_02 project=foo alias=srv_1
+   shell> iocage set notes="vmm=iocage_02 project=foo alias=srv_2" 1c11de2d
+   notes: none -> vmm=iocage_02 project=foo alias=srv_2
+   shell> iocage set notes="vmm=iocage_02 project=bar alias=srv_3" 9d94cc9e
+   notes: none -> vmm=iocage_02 project=bar alias=srv_3
+
+Update the inventory configuration. Set the option
+:ansopt:`community.general.iocage#inventory:inventory_hostname_tag` to *alias*. This tag keeps the
+value of the alias. The option :ansopt:`community.general.iocage#inventory:get_properties` must be
+enabled:
+
+.. code-block:: console
+
+   shell> cat hosts/02_iocage.yml
```

Comment:
Instead of having two code blocks (one for command line, one for file content), I would write this as:
```suggestion
enabled. For example, ``hosts/02_iocage.yml`` contains:
```